samples/snippets/src/main/java/com/examples/documentai/ParseForm.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.examples.documentai;
+
+// [START document_quickstart]
+
+import com.google.cloud.documentai.v1beta2.Document;
+import com.google.cloud.documentai.v1beta2.DocumentUnderstandingServiceClient;
+import com.google.cloud.documentai.v1beta2.FormExtractionParams;
+import com.google.cloud.documentai.v1beta2.GcsSource;
+import com.google.cloud.documentai.v1beta2.InputConfig;
+import com.google.cloud.documentai.v1beta2.KeyValuePairHint;
+import com.google.cloud.documentai.v1beta2.ProcessDocumentRequest;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class ParseForm {
+  public static void parseForm() throws IOException, ExecutionException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region";    // available regions https://cloud.google.com/compute/docs/regions-zones
```

Comment:
That comment is for the compute regions, which don't relate to the API's available regions. 

Instead let's just use `us-central1` and inline this variable.

Reply:
Okay